paper/paper.Rmd

```diff
@@ -23,15 +23,24 @@ knitr::opts_chunk$set(echo = TRUE)
 
 # Summary 
 
-Data are increasingly wider; in modern genomics, for example, high-resolution genetic data captures much more information than it did just a decade ago. While improved measurements contribute immensely to science, they also increase computational burden and complicate interpretability [@karczewski2018]. Data reduction techniques, such as principal components analysis and k-means clustering, are vital tools used to address these issues, particularly for noise and redundancy. However, these techniques may lead to problems in scalability, information loss, and interpretability [@malod-dogin-2018]. The Partition framework is an approach to data reduction that is flexible, scalable, and interpretable, developed to address information loss while maintaining speed[@Millstein2019].
+Data are increasingly feature-rich, including many variables for each observation; in modern genomics, for example, high-resolution genetic data captures much more information than it did just a decade ago. While improved measurements contribute immensely to science, they also increase computational burden and complicate interpretability [@karczewski2018]. Data reduction techniques, such as principal components analysis (PCA) and K-Means clustering, are vital tools used to address these issues, particularly for noise and redundancy. However, these techniques may lead to problems in scalability, information loss, and interpretability [@malod-dogin-2018]. The Partition framework is an approach to data reduction that is flexible, scalable, and interpretable, developed to address information loss while maintaining speed [@Millstein2019]. As opposed to other data reduction strategies, Partition only clusters data if the data reduction retains a certain amount of information. This framework is also agnostic to how clusters form; users can easily use other tools such as PCA and t-Distributed Stochastic Neighbor Embedding (t-SNE) to reduce data while constraining information loss.
```

Comment:
"As opposed to other data reduction strategies, Partition only clusters data if the data reduction retains a certain amount of information." change to: "As opposed to other data reduction strategies, Partition will only reduce data if a specified amount of information is retained."
Also: "This framework is also agnostic to how clusters form;" change to "This framework is also agnostic to how partitions are formed;"
Also: "users can easily use other tools such as PCA and t-Distributed Stochastic Neighbor Embedding (t-SNE) to reduce data while constraining information loss." change to "users can easily use other tools such as PCA and t-Distributed Stochastic Neighbor Embedding (t-SNE) to create a partition or summarize data within a partition subset and thus reduce data while constraining information loss."